src/legacy/server/saved_objects/schema/schema.ts

```diff
@@ -17,9 +17,17 @@
  * under the License.
  */
 
+import { SavedObject } from '../service';
+
 interface SavedObjectsSchemaTypeDefinition {
   isNamespaceAgnostic: boolean;
   hidden?: boolean;
+  isImportableAndExportable?: boolean;
+  titleSearchField?: string;
```

Comment:
What do you think about calling this `defaultSearchField`? I like this name because it's analogous to Elasticsearch's [`default_field` property on Query String Query](https://www.elastic.co/guide/en/elasticsearch/reference/current/query-dsl-query-string-query.html).

Reply:
I'm in!